resources/app/log-collector.yaml

```diff
@@ -0,0 +1,146 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: log-collector
+  namespace: kube-system
+  labels:
+    app: log-collector
+spec:
+  type: ClusterIP
+  ports:
+    - name: api
+      protocol: TCP
+      port: 8083
+  selector:
+    app: log-collector
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: log-forwarders
+  namespace: kube-system
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: log-collector
+  namespace: kube-system
+  labels:
+    app: log-collector
+data:
+  adapter.json: |
+    {
+      "Gravity": {
+        "ApiListenAddr": "0.0.0.0:8083",
+        "Kubernetes": {
+          "Namespace": "kube-system",
+          "ForwarderConfigMapName": "log-forwarders"
+        }
+      },
+
+      "Logrange": {
+        "Partition": "logrange.pipe=__default__",
+        "ForwarderTmplFile": "/opt/logrange/gravity/config/forward-tmpl.json",
+        "CronQueries": [
+          {
+            "Query": "truncate minsize 4mb maxsize 500mb maxdbsize 10gb",
+            "PeriodSec": 60
```

Comment:
Isn't that a bit too aggressive schedule for truncating job (just wondering)?

Reply:
Mmm really it depends on how fast the logs are written (to keep them within the bounds)... But in general this doesn't add any obligations in terms of 'global' locks (we don't have this) or anything like that...

Reply:
I think 60 sec is fine

Reply:
Okay, cool.